lib/web/password.go

```diff
@@ -39,7 +39,7 @@ type changePasswordReq struct {
 	// SecondFactorToken is user 2nd factor token
 	SecondFactorToken string `json:"second_factor_token"`
 	// WebauthnAssertionResponse is a Webauthn response
-	WebauthnAssertionResponse *wantypes.CredentialAssertionResponse `json:"webauthnAssertionResponse"`
+	WebauthnAssertionResponse *wantypes.CredentialAssertionResponse `json:"webauthn_assertion_response"`
```

Comment:
Why the tag change here? This is a breaking change.

Reply:
I just spotted an inconsistency and thought it's not a breaking change if it only affects the web API, which is being updated in the same PR.

Reply:
Discussed out-of-band, whatever you prefer is fine.

Reply:
This is something we need to address eventually with the web team. This is a breaking change if our web client hits a proxy of a different version. As of now, we haven't seen too many of these issues, but when customers who run multiple proxies in a load balancer, this will break (or leaf clusters too i think?). It isn't that frequent, but frequent enough that we have a quarter goal this quarter to be able to serve the web asset bundle to different versions.

I would suggest not including this line

Reply:
+1 please avoid the breaking change.